### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by webbyfox**

### CODE REVIEW

File: .coverage [hunk 1]
```diff
@@ -0,0 +1 @@
...28],[28,-26],[-22,24],[24,-22],[-33,34],[34,-33]],"/var/www/fil/djangocms/.venv/src/djangocms-internalsearch/djangocms_contentexpiry/contrib/filer/__init__.py":[[-1,1],[1,-1]],"/var/www/fil/djangocms/.venv/src/djangocms-internalsearch/djangocms_contentexpiry/contrib/filer/internal_search.py":[[-1,1],[1,2],[2,3],[3,5],[5,-1]],"/var/www/fil/djangocms/.venv/src/djangocms-internalsearch/djangocms_contentexpiry/backends/__init__.py":[[-1,1],[1,-1]],"/var/www/fil/djangocms/.venv/src/djangocms-internalsearch/djangocms_contentexpiry/backends/elasticsearch2.py":[[-1,1],[1,3],[3,6],[-6,6],[6,7],[7,-6],[6,-1]],"/var/www/fil/djangocms/.venv/src/djangocms-internalsearch/djangocms_contentexpiry/unified_index.py":[[-1,1],[1,3],[3,6],[-6,6],[6,8],[8,-6],[6,-1],[-8,9],[9,10],[10,11],[11,13],[13,14],[14,15],[15,16],[16,17],[17,20],[20,13],[13,21],[21,-8]],"/var/www/fil/djangocms/.venv/src/djangocms-internalsearch/djangocms_contentexpiry/backends/whoosh.py":[[-1,1],[1,3],[3,6],[-6,6],[6,7],[7,-6],[6,-1]],"/var/www/fil/djangocms/.venv/src/djangocms-internalsearch/djangocms_contentexpiry/cms_toolbars.py":[[-1,1],[1,3],[3,4],[4,6],[6,9],[9,10],[10,15],[15,16],[-15,15],[15,19],[19,20],[20,-15],[16,-1]]}}
```
Comment (chaosk) [inline .coverage]: Remove this file and add to .gitignore

File: LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2017, Divio AG
```
Comment (chaosk) [inline LICENSE.txt]: Change this (year, company)

File: djangocms_contentexpiry/migrations/0001_initial.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ContentExpiry',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('activation_date', models.DateTimeField(auto_now_add=True)),
+                ('expiry_date', models.DateTimeField(default=datetime.datetime(2019, 1, 2, 8, 33, 19, 526685))),
```
Comment (chaosk) [inline djangocms_contentexpiry/migrations/0001_initial.py]: Doesn't seem right, check comment below

File: djangocms_contentexpiry/models.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class ContentExpiry(models.Model):
+    activation_date = models.DateTimeField(auto_now_add=True)
+    expiry_date = models.DateTimeField(
+        default=datetime.now() + timedelta(days=30)
```
Comment (chaosk) [inline djangocms_contentexpiry/models.py]: This is a value that's calculated once on application startup. Need to make it a separate function or wrap it in `lambda: `. Also use `django.utils.timezone.now` instead.

File: djangocms_contentexpiry/models.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    )
+    modified_by =  models.ForeignKey(
```
Comment (chaosk) [inline djangocms_contentexpiry/models.py]:
```suggestion
    modified_by = models.ForeignKey(
```

File: djangocms_contentexpiry/models.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+from django.conf import settings
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class ContentExpiry(models.Model):
+    activation_date = models.DateTimeField(auto_now_add=True)
```
Comment (chaosk) [inline djangocms_contentexpiry/models.py]: Why is it called `activation_date`?

File: setup.py [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    long_description=open('README.md').read(),
+    classifiers=[
+        'Framework :: Django',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
+        'Operating System :: OS Independent',
+        'Topic :: Software Development'
+    ],
+    install_requires=INSTALL_REQUIREMENTS,
+    author='Rizwan Mansuri',
```
Comment (chaosk) [inline setup.py]: Check version-locking's setup.py

**Review by chaosk — CHANGES_REQUESTED** [review]

File: djangocms_contentexpiry/models.py (hunk 6 above)
Comment (webbyfox) [reply djangocms_contentexpiry/models.py]: No idea, It was in the story, basically its creation date of expiry object.

**Review by chaosk — APPROVED** [review]

**Pull request merged**